lcviz/parsers.py

```diff
@@ -22,6 +22,10 @@ def light_curve_parser(app, file_obj, data_label=None, show_in_viewer=True, **kw
     elif isinstance(file_obj, lightkurve.LightCurve):
         light_curve = file_obj
 
+    # load a LightCurve object:
+    elif isinstance(file_obj, lightkurve.targetpixelfile.KeplerTargetPixelFile):
+        light_curve = file_obj
```

Comment:
Is there any reason we can't merge these lines?  

```suggestion
    elif isinstance(file_obj, (lightkurve.LightCurve, lightkurve.KeplerTargetPixelFile):
        light_curve = file_obj
```

How about checking against the parent `TargetPixelFile` class (here and in utils as well) - or is some of this logic specific to only support Kepler?

Reply:
As far as I can tell, there is only one feature (quality bitmask conversion) that is specific to the mission within the TPF translator. I can put in a check for the mission to use the correct bitmask converter, and then this should (hopefully) work for all TPF classes.